apps/api/src/utils/email_handler.py

```diff
@@ -10,8 +10,8 @@
     Template,
 )
 
-IH_SENDER = ("apply@irvinehacks.com", "IrvineHacks 2024 Applications")
-REPLY_TO_HACK_AT_UCI = ("hack@uci.edu", "Hack at UCI")
+IH_SENDER = ("apply@irvinehacks.com", "IrvineHacks 2025 Applications")
+REPLY_TO_HACK_AT_UCI = ("irvinehacks2025@gmail.com", "Hack at UCI")
```

Comment:
Thought: if we set up forwarding from `apply@irvinehacks.com` to the Gmail address, I think we could get away with not specifying a reply-to address and have the default of replying to the sender.